Resources/Prototypes/Entities/Markers/Spawners/Random/materials.yml

```
@@ -35,13 +26,10 @@
         - state: red
         - sprite: Objects/Materials/materials.rsi
           state: wood_2
-    - type: RandomSpawner
-      prototypes:
-        - MaterialWoodPlank10
-        - MaterialCardboard10
-        - PartRodMetal10
-      chance: 0.8
-      offset: 0.0
+    - type: EntityTableSpawner
+      table: !type:NestedSelector
+        tableId: RandomAllSupplementaryMaterialsTable
+        prob: 0.9
```

```suggestion
        prob: 0.8
```
keep the old value, no need to make balance changes in this PR

---

whoops mustve left it by mistake